scripts/icons.js

```diff
@@ -0,0 +1,234 @@
+/*
+Copyright © 2021 The Sage Group plc or its licensors. All Rights reserved
+ */
+
+const Figma = require('figma-js')
+const fetch = require('node-fetch')
+const Handlebars = require('handlebars')
+const {
+  outputFile,
+  outputJson,
+  removeSync,
+  ensureDir,
+  readFileSync
+} = require('fs-extra')
+const { generateFonts } = require('fantasticon')
+const {
+  join,
+  relative,
+  resolve,
+  dirname,
+  sep,
+  posix
+} = require('path')
+const kebabCase = require('lodash/kebabCase')
+const pick = require('lodash/pick')
+
+const uniqueValues = require('./utils/unique-values')
+const collect = require('./utils/collect')
+
+async function getIconsArray (personalAccessToken, fileId) {
+  console.log(`Fetching icon components information from Figma file ${fileId}...`)
+  const client = Figma.Client({ personalAccessToken })
+
+  const components = await client.file(fileId).then((file) => {
+    console.log(`  File name: ${file.data.name}`)

```

Comment:
In several places for e.g. file.data, icon, config you could do destructuring. I understand that you did not use this to maintain better readability?

Reply:
No, I just haven't notice that :D